src/main/java/frc/robot/commands/PoseTransform.java

```diff
@@ -0,0 +1,72 @@
+package frc.robot.commands;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.TestConstants;
+import frc.robot.subsystems.Drivetrain;
+
+/**
+ * Tests the odometry of the robot by driving a certain distance and calculating the error.
+ */
+public class PoseTransform extends CommandBase {
+
```

Comment:
See my comment on the other command. This seems to be the same command as odometerTest. Just make one command with different constructors.

Reply:
Ok I looked at it now and it looks like one is a test and the other is a actual command. I would just make the test call the actual one. And, you only need 1 test command, one actual command